bindings/wrap/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "tiktoken-wrap"
+version = "0.1.0"
+description = "A Polywrap Wasm wrapper of tiktoken-rs"
+authors = ["Polywrap"]
+repository = "https://github.com/polywrap/monorepo"
```

Comment:
Should be `https://github.com/polywrap/tiktoken-rs`.

Also @Niraj-Kamdar let's merge these changes into our fork's main branch so that when you view the repo you see our latest version. Currently the PR we have to the maintainer's repo is from a branch you've created.